ios/PlayerModule.swift

```diff
@@ -450,4 +462,39 @@ class PlayerModule: NSObject, RCTBridgeModule {
             resolve(self?.players[nativeId]?.isAd)
         }
     }
+    
+    /**
+     The current time shift of the live stream in seconds. This value is always 0 if the active `source` is not a
+     live stream or there is no active playback session.
```

Comment:
We don't have the concept of playback session on iOS.

```suggestion
     The current time shift of the live stream in seconds. This value is always 0 if the active `source` is not a
     live stream or there are no sources loaded.
```